code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm

```diff
@@ -3,20 +3,107 @@
 	plasma_cost = 75
 	acid_type = /obj/effect/xenomorph/acid/weak
 
-/datum/action/xeno_action/activable/salvage_plasma
-	name = "Salvage Plasma"
+/datum/action/xeno_action/activable/salvage_biomass
+	name = "Salvage Biomass"
 	action_icon_state = "salvage_plasma"
-	ability_name = "salvage plasma"
-	var/plasma_salvage_amount = PLASMA_SALVAGE_AMOUNT
-	var/salvage_delay = 3 SECONDS
-	var/max_range = 1
+	ability_name = "salvage biomass"
 	keybind_signal = COMSIG_XENOABILITY_SALVAGE_PLASMA
```

Comment:
Don't mess with the keybind signals just now, but please open a new PR that changes these to the proper nomenclature in the near future.

Reply:
That's the idea. For now it'll work as I'm just plugging the keybind into that which already exists.